src/redux/reducers/home.js

```diff
@@ -0,0 +1,27 @@
+const initialState = {
+  members: 0,
+  repos: 0,
+  teams: 0,
+  projects: 0,
+  status: 0,
+};
+
+export default (state = initialState, action) => {
+  const { type, payload } = action;
+  switch(type) {
+  case 'HOMESTATS_PENDING':
+    return {
+      status: 1,
+    }
+  case 'HOMESTATS_FINISH':
+    return {
+      ...payload,
```

Comment:
To avoid missing out some necessary state, which could break the app. Change this to:
```
{
...state,
...payload
}
```

Reply:
Thanks @abayo-luc , I forget to watch out on it